modules/extensions/partials/proc_java-lombok.adoc

```diff
@@ -0,0 +1,46 @@
+[id="proc_java-lombok_{context}"]
+= Java Lombok
+
+[role="_abstract"]
+This section shows how to enable the Lombok support in your Java projects.
+
+Project Lombok is a Java library that automatically plugs into your editor and build tools, spicing up your Java.
+Never write another getter or equals method again, with one annotation your class has a fully featured builder, Automate your logging variables, and much more.
```

Comment:
Not sure if we need this line, it might be better to just provide a link to the Lombok homepage -- let them sell the project.

Reply:
OK I will remove these lines.